ERROR - 2022-12-02 09:52:48 --> mysqli_sql_exception: Column 'jadwal_tes_id' cannot be null in /home/sunanpan/public_html/psb/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /home/sunanpan/public_html/psb/system/Database/MySQLi/Connection.php(292): mysqli->query('INSERT INTO `sa...', 0)
#1 /home/sunanpan/public_html/psb/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `sa...')
#2 /home/sunanpan/public_html/psb/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `sa...')
#3 /home/sunanpan/public_html/psb/system/Database/BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `sa...', Array, false)
#4 /home/sunanpan/public_html/psb/system/Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
#5 /home/sunanpan/public_html/psb/system/BaseModel.php(749): CodeIgniter\Model->doInsert(Array)
#6 /home/sunanpan/public_html/psb/system/Model.php(667): CodeIgniter\BaseModel->insert(Array, true)
#7 /home/sunanpan/public_html/psb/app/Controllers/FormPrestasiController.php(173): CodeIgniter\Model->insert(Array)
#8 /home/sunanpan/public_html/psb/system/CodeIgniter.php(927): App\Controllers\FormPrestasiController->simpanPrestasi()
#9 /home/sunanpan/public_html/psb/system/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\FormPrestasiController))
#10 /home/sunanpan/public_html/psb/system/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /home/sunanpan/public_html/psb/index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-12-02 10:37:40 --> mysqli_sql_exception: Column 'jadwal_tes_id' cannot be null in /home/sunanpan/public_html/psb/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /home/sunanpan/public_html/psb/system/Database/MySQLi/Connection.php(292): mysqli->query('INSERT INTO `sa...', 0)
#1 /home/sunanpan/public_html/psb/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `sa...')
#2 /home/sunanpan/public_html/psb/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `sa...')
#3 /home/sunanpan/public_html/psb/system/Database/BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `sa...', Array, false)
#4 /home/sunanpan/public_html/psb/system/Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
#5 /home/sunanpan/public_html/psb/system/BaseModel.php(749): CodeIgniter\Model->doInsert(Array)
#6 /home/sunanpan/public_html/psb/system/Model.php(667): CodeIgniter\BaseModel->insert(Array, true)
#7 /home/sunanpan/public_html/psb/app/Controllers/FormPrestasiController.php(173): CodeIgniter\Model->insert(Array)
#8 /home/sunanpan/public_html/psb/system/CodeIgniter.php(927): App\Controllers\FormPrestasiController->simpanPrestasi()
#9 /home/sunanpan/public_html/psb/system/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\FormPrestasiController))
#10 /home/sunanpan/public_html/psb/system/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /home/sunanpan/public_html/psb/index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-12-02 12:36:48 --> Argument 1 passed to Myth\Auth\Models\GroupModel::getGroupsForUser() must be of the type int, null given, called in /home/sunanpan/public_html/psb/app/Controllers/Admin/DashboardController.php on line 15
in FCPATH/vendor/myth/auth/src/Models/GroupModel.php on line 95.
 1 APPPATH/Controllers/Admin/DashboardController.php(15): Myth\Auth\Models\GroupModel->getGroupsForUser(null)
 2 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Admin\DashboardController->cekredirect()
 3 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\DashboardController))
 4 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-02 13:03:30 --> mysqli_sql_exception: Column 'jadwal_tes_id' cannot be null in /home/sunanpan/public_html/psb/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /home/sunanpan/public_html/psb/system/Database/MySQLi/Connection.php(292): mysqli->query('INSERT INTO `sa...', 0)
#1 /home/sunanpan/public_html/psb/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `sa...')
#2 /home/sunanpan/public_html/psb/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `sa...')
#3 /home/sunanpan/public_html/psb/system/Database/BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `sa...', Array, false)
#4 /home/sunanpan/public_html/psb/system/Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
#5 /home/sunanpan/public_html/psb/system/BaseModel.php(749): CodeIgniter\Model->doInsert(Array)
#6 /home/sunanpan/public_html/psb/system/Model.php(667): CodeIgniter\BaseModel->insert(Array, true)
#7 /home/sunanpan/public_html/psb/app/Controllers/FormPrestasiController.php(173): CodeIgniter\Model->insert(Array)
#8 /home/sunanpan/public_html/psb/system/CodeIgniter.php(927): App\Controllers\FormPrestasiController->simpanPrestasi()
#9 /home/sunanpan/public_html/psb/system/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\FormPrestasiController))
#10 /home/sunanpan/public_html/psb/system/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /home/sunanpan/public_html/psb/index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-12-02 14:28:13 --> mysqli_sql_exception: Column 'jadwal_tes_id' cannot be null in /home/sunanpan/public_html/psb/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /home/sunanpan/public_html/psb/system/Database/MySQLi/Connection.php(292): mysqli->query('INSERT INTO `sa...', 0)
#1 /home/sunanpan/public_html/psb/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `sa...')
#2 /home/sunanpan/public_html/psb/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `sa...')
#3 /home/sunanpan/public_html/psb/system/Database/BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `sa...', Array, false)
#4 /home/sunanpan/public_html/psb/system/Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
#5 /home/sunanpan/public_html/psb/system/BaseModel.php(749): CodeIgniter\Model->doInsert(Array)
#6 /home/sunanpan/public_html/psb/system/Model.php(667): CodeIgniter\BaseModel->insert(Array, true)
#7 /home/sunanpan/public_html/psb/app/Controllers/FormPrestasiController.php(173): CodeIgniter\Model->insert(Array)
#8 /home/sunanpan/public_html/psb/system/CodeIgniter.php(927): App\Controllers\FormPrestasiController->simpanPrestasi()
#9 /home/sunanpan/public_html/psb/system/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\FormPrestasiController))
#10 /home/sunanpan/public_html/psb/system/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /home/sunanpan/public_html/psb/index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-12-02 16:01:55 --> mysqli_sql_exception: Column 'jadwal_tes_id' cannot be null in /home/sunanpan/public_html/psb/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /home/sunanpan/public_html/psb/system/Database/MySQLi/Connection.php(292): mysqli->query('INSERT INTO `sa...', 0)
#1 /home/sunanpan/public_html/psb/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `sa...')
#2 /home/sunanpan/public_html/psb/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `sa...')
#3 /home/sunanpan/public_html/psb/system/Database/BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `sa...', Array, false)
#4 /home/sunanpan/public_html/psb/system/Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
#5 /home/sunanpan/public_html/psb/system/BaseModel.php(749): CodeIgniter\Model->doInsert(Array)
#6 /home/sunanpan/public_html/psb/system/Model.php(667): CodeIgniter\BaseModel->insert(Array, true)
#7 /home/sunanpan/public_html/psb/app/Controllers/FormPrestasiController.php(173): CodeIgniter\Model->insert(Array)
#8 /home/sunanpan/public_html/psb/system/CodeIgniter.php(927): App\Controllers\FormPrestasiController->simpanPrestasi()
#9 /home/sunanpan/public_html/psb/system/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\FormPrestasiController))
#10 /home/sunanpan/public_html/psb/system/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /home/sunanpan/public_html/psb/index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-12-02 22:56:48 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '* IS NULL' at line 4 in /home/sunanpan/public_html/psb/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /home/sunanpan/public_html/psb/system/Database/MySQLi/Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 /home/sunanpan/public_html/psb/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 /home/sunanpan/public_html/psb/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 /home/sunanpan/public_html/psb/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /home/sunanpan/public_html/psb/app/Controllers/Admin/SantriController.php(213): CodeIgniter\Database\BaseBuilder->get()
#5 /home/sunanpan/public_html/psb/system/CodeIgniter.php(927): App\Controllers\Admin\SantriController->detailSiswa('MA', '235')
#6 /home/sunanpan/public_html/psb/system/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\SantriController))
#7 /home/sunanpan/public_html/psb/system/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /home/sunanpan/public_html/psb/index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-02 22:56:48 --> Invalid file: admin/santri_detail_MA.php
in SYSTEMPATH/Exceptions/FrameworkException.php on line 33.
 1 SYSTEMPATH/View/View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('admin/santri_detail_MA.php')
 2 SYSTEMPATH/Common.php(1134): CodeIgniter\View\View->render('admin/santri_detail_MA', [], true)
 3 APPPATH/Controllers/Admin/SantriController.php(219): view('admin/santri_detail_MA', [...])
 4 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Admin\SantriController->detailSiswa('MA', '235')
 5 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\SantriController))
 6 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-02 23:12:14 --> mysqli_sql_exception: Column 'jadwal_tes_id' cannot be null in /home/sunanpan/public_html/psb/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /home/sunanpan/public_html/psb/system/Database/MySQLi/Connection.php(292): mysqli->query('INSERT INTO `sa...', 0)
#1 /home/sunanpan/public_html/psb/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `sa...')
#2 /home/sunanpan/public_html/psb/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `sa...')
#3 /home/sunanpan/public_html/psb/system/Database/BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `sa...', Array, false)
#4 /home/sunanpan/public_html/psb/system/Model.php(279): CodeIgniter\Database\BaseBuilder->insert()
#5 /home/sunanpan/public_html/psb/system/BaseModel.php(749): CodeIgniter\Model->doInsert(Array)
#6 /home/sunanpan/public_html/psb/system/Model.php(667): CodeIgniter\BaseModel->insert(Array, true)
#7 /home/sunanpan/public_html/psb/app/Controllers/FormPrestasiController.php(173): CodeIgniter\Model->insert(Array)
#8 /home/sunanpan/public_html/psb/system/CodeIgniter.php(927): App\Controllers\FormPrestasiController->simpanPrestasi()
#9 /home/sunanpan/public_html/psb/system/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\FormPrestasiController))
#10 /home/sunanpan/public_html/psb/system/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /home/sunanpan/public_html/psb/index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2022-12-02 23:16:01 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '* IS NULL' at line 4 in /home/sunanpan/public_html/psb/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /home/sunanpan/public_html/psb/system/Database/MySQLi/Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 /home/sunanpan/public_html/psb/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 /home/sunanpan/public_html/psb/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 /home/sunanpan/public_html/psb/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /home/sunanpan/public_html/psb/app/Controllers/Admin/SantriController.php(213): CodeIgniter\Database\BaseBuilder->get()
#5 /home/sunanpan/public_html/psb/system/CodeIgniter.php(927): App\Controllers\Admin\SantriController->detailSiswa('MTs', '259')
#6 /home/sunanpan/public_html/psb/system/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\SantriController))
#7 /home/sunanpan/public_html/psb/system/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /home/sunanpan/public_html/psb/index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-12-02 23:41:59 --> Error connecting to the database: mysqli_sql_exception: No such file or directory in /home/sunanpan/public_html/psb/system/Database/MySQLi/Connection.php:171
Stack trace:
#0 /home/sunanpan/public_html/psb/system/Database/MySQLi/Connection.php(171): mysqli->real_connect('localhost', 'sunanpan_new_ps...', '@Sunanpandanara...', 'sunanpan_new_ps...', 3306, '', 0)
#1 /home/sunanpan/public_html/psb/system/Database/BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /home/sunanpan/public_html/psb/system/Database/BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/sunanpan/public_html/psb/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /home/sunanpan/public_html/psb/system/Model.php(182): CodeIgniter\Database\BaseBuilder->get()
#5 /home/sunanpan/public_html/psb/system/BaseModel.php(524): CodeIgniter\Model->doFind(true, '6')
#6 /home/sunanpan/public_html/psb/vendor/myth/auth/src/Authentication/AuthenticationBase.php(130): CodeIgniter\BaseModel->find('6')
#7 /home/sunanpan/public_html/psb/vendor/myth/auth/src/Authentication/LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
#8 /home/sunanpan/public_html/psb/vendor/myth/auth/src/Filters/RoleFilter.php(21): Myth\Auth\Authentication\LocalAuthenticator->check()
#9 /home/sunanpan/public_html/psb/system/Filters/Filters.php(173): Myth\Auth\Filters\RoleFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#10 /home/sunanpan/public_html/psb/system/CodeIgniter.php(455): CodeIgniter\Filters\Filters->run('dashboard', 'before')
#11 /home/sunanpan/public_html/psb/system/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /home/sunanpan/public_html/psb/index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No such file or directory in /home/sunanpan/public_html/psb/system/Database/MySQLi/Connection.php:215
Stack trace:
#0 /home/sunanpan/public_html/psb/system/Database/BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /home/sunanpan/public_html/psb/system/Database/BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/sunanpan/public_html/psb/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /home/sunanpan/public_html/psb/system/Model.php(182): CodeIgniter\Database\BaseBuilder->get()
#4 /home/sunanpan/public_html/psb/system/BaseModel.php(524): CodeIgniter\Model->doFind(true, '6')
#5 /home/sunanpan/public_html/psb/vendor/myth/auth/src/Authentication/AuthenticationBase.php(130): CodeIgniter\BaseModel->find('6')
#6 /home/sunanpan/public_html/psb/vendor/myth/auth/src/Authentication/LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
#7 /home/sunanpan/public_html/psb/vendor/myth/auth/src/Filters/RoleFilter.php(21): Myth\Auth\Authentication\LocalAuthenticator->check()
#8 /home/sunanpan/public_html/psb/system/Filters/Filters.php(173): Myth\Auth\Filters\RoleFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), Array)
#9 /home/sunanpan/public_html/psb/system/CodeIgniter.php(455): CodeIgniter\Filters\Filters->run('dashboard', 'before')
#10 /home/sunanpan/public_html/psb/system/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /home/sunanpan/public_html/psb/index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-12-02 23:41:59 --> Unable to connect to the database.
Main connection [MySQLi]: No such file or directory
in SYSTEMPATH/Database/BaseConnection.php on line 418.
 1 SYSTEMPATH/Database/BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `users`.`deleted_at` IS NULL
AND `users`.`id` = :users.id:', [...], false)
 3 SYSTEMPATH/Model.php(182): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(524): CodeIgniter\Model->doFind(true, '6')
 5 FCPATH/vendor/myth/auth/src/Authentication/AuthenticationBase.php(130): CodeIgniter\BaseModel->find('6')
 6 FCPATH/vendor/myth/auth/src/Authentication/LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
 7 FCPATH/vendor/myth/auth/src/Filters/RoleFilter.php(21): Myth\Auth\Authentication\LocalAuthenticator->check()
 8 SYSTEMPATH/Filters/Filters.php(173): Myth\Auth\Filters\RoleFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), [...])
 9 SYSTEMPATH/CodeIgniter.php(455): CodeIgniter\Filters\Filters->run('dashboard', 'before')
10 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
